Caches/test/EteTests/testRunner.py

```diff
@@ -29,16 +29,13 @@ def findDiff(first, second):
     return chr(c)
  
 def chechTestResult(goldenPath, testResult):
```

Comment:
Could you please also fix misprint in the function name: chechTestResult -> checkTestResult?